.gitignore

```diff
@@ -66,6 +66,8 @@ php-cs-fixer
 translations/*
 mails/*
 !mails/en/
+!mails/themes/
```

Comment:
this line is unneeded because of the line below

Reply:
I don't follow you
All mails folder are ignored, except `en` (already present) and this new `themes` folder
Or maybe you're talking about `!mails/themes/*` ?

Reply:
Yep i'm taking about `!mails/themes/*`